docs/resources/aws_stepfunctions_state_machine.md

```diff
@@ -0,0 +1,87 @@
+---
+title: About the aws_stepfunctions_state_machine Resource
+platform: aws
+---
+
+# aws\_stepfunctions\_state\_machine
+
+Use the `aws_stepfunctions_state_machine` InSpec audit resource to test properties of a single specific Step Functions state machine.
+
+A state machine consists of a collection of states that can do work (Task states), determine which states to transition next (Choice states), stop an execution with an error (Fail states), and so on.
+
+## Syntax
+
+Ensure that an identity pool exists.
+
+    describe aws_stepfunctions_state_machine(state_machine_arn: 'STATE_MACHINE_ARN') do
+      it { should exist }
+    end
+
+## Parameters
+
+`state_machine_arn` _(required)_
+
+The ARN of the state machine.
+
+For additional information, see the [AWS documentation on Step Functions state machine](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-stepfunctions-statemachine.html).
+
+## Properties
+
+| Property | Description|
+| --- | --- |
+| state_machine_arn | The ARN of the state machine. |
+| name | The name of the state machine. |
+| status | The current status of the state machine. |
+| definition | The Amazon States Language definition of the state machine. |
+| role_arn | The ARN of the IAM role used when creating this state machine. |
+| type | The type of the state machine. Valid values: `STANDARD` or `EXPRESS`. |
+| creation_date | The creation date of the state machine. |
+| logging_configuration (level) | The category of execution history in which events are logged. |
+| logging_configuration (include_execution_data) | Whether execution data is included in your log. Valid values: `true` or `false`. |
+| logging_configuration (destinations) | An array of objects that describes where your execution history events will be logged. |
+| logging_configuration (enabled) | Selects whether or not the state machine's AWS X-Ray tracing is enabled. Valid values: `true` or `false`. |
```

Comment:
Should `logging_configuration (enabled)` be `tracing_configuration (enabled)` ?

Reply:
Yeah it should be tracing_configuration. I corrected it.